Assets/MixedRealityToolkit.Services/InputSystem/Editor/MixedRealityCanvasInspector.cs

```diff
@@ -341,5 +357,26 @@ public override void OnInspectorGUI()
 
             serializedObject.ApplyModifiedProperties();
         }
+
+        private static List<Graphic> GetGraphicsWhichRequireScaleMeshEffect(Object[] targets)
+        {
+            var output = new List<Graphic>();
```

Comment:
just for fun, you could consider implementing this using LINQ.

using System.Linq;

targets.SelectMany(target => (target as Canvas).GetComponentsInChildren<Graphic>())
.Where(graphic =>StandardShaderUtility.IsUsingMrtkStandardShader(graphic.material) && graphic.GetComponent<ScaleMeshEffect>() == null);

Reply:
PS.  If you do that, then this function could return IEnumerable<Graphic> instead of List<Graphic>

---
In reply to: [321028562](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5882#discussion_r321028562) [](ancestors = 321028562)

Reply:
Haha, I haven't heard many people use the work fun and LINQ in the same sentence. ;) I do like the idea of returning an IEnumerable. I'll fix that!